pinot-spi/src/main/java/org/apache/pinot/spi/config/table/assignment/InstanceReplicaGroupPartitionConfig.java

```diff
@@ -41,7 +41,8 @@
   @JsonPropertyDescription("Number of partitions for replica-group based selection, do not partition the replica-group (1 partition) if not specified")
   private final int _numPartitions;
 
-  @JsonPropertyDescription("Number of instances per partition (within a replica-group) for replica-group based selection, select all instances if not specified")
+  @JsonPropertyDescription(
+      "Number of instances per partition (within a replica-group) for replica-group based selection, select all instances if not specified")
```

Comment:
This line is still too long?

Reply:
Although the error msg shows don't go beyond 120, the actual limit is 160 :p